tagstudio/src/qt/widgets/preview_panel.py

```diff
@@ -535,7 +534,7 @@ def update_widgets(self):
                         DecompressionBombError,
                     ) as e:
                         self.dimensions_label.setText(
-                            f"{extension.upper()}  •  {format_size(os.stat(filepath).st_size)}"
+                            f"{filepath.suffix.upper()}  •  {format_size(os.stat(filepath).st_size)}"
```

Comment:
```suggestion
                            f"{filepath.suffix.upper()[1:]}  •  {format_size(os.stat(filepath).st_size)}"
```
Removes leading "." from filetype display (wish I could comment on all these different lines at once)